andela/ah-frontend-space
# #166880074 Enable users to reset their passwords.

#### What does this PR do?

 - Implements the password-reset functionality

#### Description of Task to be completed?

- Add a page on which to verify that a user is registered
- Call the endpoint that sends a password reset link to the user's
  email address after verification of the user's email address
- Add a page that the user is redirected to in order to enter a
  new password

#### How should this be manually tested?

- On the [sign in](https://ah-frontend-space-stagin-pr-20.herokuapp.com/signin) page, click the `forgot password?` link
- You will be redirected to the `confirm email` page where you will enter a registered email address.
- If successful, you will receive a password reset link in your email.
- Click the link to be redirected to the password reset page.
- Enter your new password twice to reset your password.

#### What are the relevant pivotal tracker stories?

[#166880074](https://www.pivotaltracker.com/story/show/166880074)

#### Screenshots (if appropriate)

<img width="1440" alt="Screenshot 2019-07-20 at 17 36 28" src="https://user-images.githubusercontent.com/6644707/61580025-ab69a080-ab15-11e9-82f5-b593da8cd35d.png">

<img width="1431" alt="Screenshot 2019-07-20 at 17 39 39" src="https://user-images.githubusercontent.com/6644707/61580027-bde3da00-ab15-11e9-8509-76003048e4a5.png">
